Sources/Fluent/Database/Driver.swift

```diff
@@ -46,4 +54,8 @@ extension Driver {
         let nodes = try values.map { try $0.makeNode() }
         return try self.raw(raw, nodes)
     }
+    
+    public var idType: Schema.Field.KeyType {
+        return .int
```

Comment:
I added this in an extension – on reflection, not sure if this should be there. May be better for all driver implementations to knowingly set it? given that this concept of an idType is new and some drivers may not be behaving correctly and the compiler doesn't inform you about it being necessary to declare this.